docs/dispatcher/router.rst

```diff
@@ -12,13 +12,13 @@ Event observers
 
 .. warning::
 
-    All handlers is always should be an asynchronous.
-    Name of handler function is not important. Event argument name is also is not important but is recommended to don't overlap the name with contextual data in due to function can not accept two arguments with the same name.
+    All handlers always should be asynchronous.
+    The name of the handler function is not important. The event argument name is also not important but it is recommended to not overlap the name with contextual data in due to function can not accept two arguments with the same name.
 
-Here is list of available observers and examples how to register handlers (In examples used only @decorator-style):
-
-For examples used decorator-style registering handlers but if you dont like @decorators just use :obj:`<event type>.register(...)` method instead.
+Here is the list of available observers and examples of how to register handlers (In examples used only @decorator-style):
 
+In these examples only decorator-style registering handlers are used, but if you dont like @decorators just use :obj:`<event type>.register(...)` method instead
+.
```

Comment:
Please, remove turn `.` back to previous sentence. And turn back the empty string before header

Reply:
Done